### ISSUE TITLE
[Proposal] Restriction duration rework

### ISSUE THREAD

**Issue by Kverto** [ppy/osu-wiki#1936]
As peppy suggested a few weeks ago, I’d like to put forward my proposal on changing some of the ways cheaters are dealt with up for discussion. My aim is to polish my proposal with the help of a constructive discussion here, which would result in a PR and ultimately in tangible changes in how restrictions are dealt with. Excuse me with my inexperience with github in advance, I’m still learning how things work around here. 

First I’d like to propose a change in restriction length, as it seems like cases with various severity are not treated accordingly enough. Secondly, I would like to propose post-restriction changes to the system, such as a short, but clearly visible marker on unrestricted players’ profile to serve as a tangible consequence that signals the severity of rule breaking. This is by no means a permanent mark of shame.

Currently, most first time offenders can appeal after 3 months. This is true for players who have been cheating for months, reaping the rewards of community recognition and possible monetary gains by a twitch career to players who broke the rules on the spur of the moment and enjoyed none of the benefits that calculated rule breakers did.

Although heat of the moment cheating has to be punished, calculated rule breaking should definitely involve more severe consequences. This can be solved by **increasing the restriction duration by 1 month for every week the rule breaker hesitated to come forward**. To avoid permanent restrictions, this period can have an upper limit. 1 year feels like an appropriate ceiling, as appeals do not guarantee unrestrictions anyway.

Tournament cheaters are restricted from participating in official tournaments, including badged tournaments, which is a good step forward, but there needs to be an indicator for tournament organisers to filter out players who would void the tournament. As tournament cheating is one of the most heinous offences, I’d like to propose a [simple red text](http://puu.sh/Cpun3/aea9d1d468.jpg). Alternatively (if the text is considered too extreme) a [simple, unassuming symbol](http://puu.sh/CpuqT/a55f066da4.jpg) can represent their ineligibility to official tournaments. Once they are eligible for such tournaments again, the sign or text can be removed entirely. 

Originally, I would’ve proposed [marks of shame](https://cdn.discordapp.com/attachments/271408718808088583/512271236395696149/cheating_cross.png) to every type of rule breakers, which could have been removed after either monetary (in form of supporter) or community compensation, or even after some time automatically, but after further input, this would be too radical at the moment.

Tl;dr: Increase punishments for calculated rule breakers, make the tournament organisers’ job easier by applying public marking on ineligible players’ profile.

**Comment by WalterToro** [comment]
Even if I agree with some points I dont actually know if this is the correct place to put this proposal.
For now I will encourage you to post this on the osu! forums and discuss there unless ppy (or another team member) says otherwise.

**Comment by Ephemeralis** [comment]
There are several pragmatic considerations to keep in mind when handling restrictions:

**Increasing restriction length reduces compliance and rehabilitation.** While it seems good on paper, our experience with this over the years has conclusively shown that long-standing blockout for anything but the most severe offenses results in a flat increase in people actively attempting to evade their punishment, and seeking out more advanced services/tools to help them achieve that end. Making it difficult for people to reintegrate feeds into the syndicates of people who make their livelihood off providing these kinds of services.

**Adding extra steps of tracking/verification into the restriction appeal process causes massive, cascading slowdowns that affect all other users, and ties in to the point above.** The longer it takes us to resolve cases, the more likely people are to reoffend.

**Marking accounts to reflect their history will only increase the number of people attempting to evade punishment.** Many players who actively cheat do so for the optics of the situation - adding an openly visible 'shame' marker only disincentivizes them actively re-engaging with the community and instead positions them as a pariah, once again increasing the likelihood of these players either evading, or seeking solace in external communities of other pariah players. This creates a feedback loop which would only get worse as time goes on.

I personally feel that the way we're handling things at the moment is about the best we're going to get it. Tournament cheaters suffer huge, unseen penalties for their act as they automatically fail screening for any badge-awarding tournament (organizers are made aware of their history as well), meaning that one offense is enough to essentially gut their chances at competitive play forever, and there is currently no way for them to appeal or remove this hidden mark.

Even then, these tournament bans are subject to the shortfallings listed above. It's a difficult situation.

**Comment by Kverto** [comment]
> Increasing restriction length reduces compliance and rehabilitation.

If that was a main concern, the offenders wouldn't need to list all their wrongdoings in their appeal. From what I noticed, many players are being denied due to not being completely honest with the staff. This point is inconsistent with the current way restrictions are handled. Players wouldn't need to detail every part of their rule-breaking to get unrestricted if this was a major issue. 

> Adding extra steps of tracking/verification into the restriction appeal process causes massive, cascading slowdowns

I could understand this point, but a lot of data is already being gathered from rule breakers. The support seem to know a lot of details about how these players broke the rules. At that point, tying restriction length to the first cheated score does not seem to put any strain onto the system, as the support most likely possess the information.

Please note that the incrementally increasing punishment would create an incentive for people to come forward as soon as possible, which could actually remove some burden from the support infrastructure. Some of these players may never come forward without this incentive. It creates a sense of urgency, which the current system does not have. 

> Marking accounts to reflect their history will only increase the number of people attempting to evade punishment.

The mark I proposed here would be as unassuming as it would get and its main purpose would be to inform tournament staff which players cannot enter their badged tournaments. The symbol solution would be a compromise, so only the most informed part of the playerbase would recognise (or even notice) its meaning. It would remove the burden from the support by allowing the tournament organisers to screen the players themselves.

While I see your point that marking ex-cheaters and increasing restriction time could make their re-integration more difficult, the greatest hurdle from re-integration seem to be entirely overlooked, which is **the community's hostility towards unrestricted players who are perceived to not receive their fair share of punishment**.

The hostility stem from the extremely lenient sentencing of high profile, serious rule-breakers who managed to follow the appeal protocol (which is not very difficult if every single one of their offenses are publicly scrutinised). This had led to players reacting to any high profile restrictions with "see you in 4 months", which clearly shows that even if these players will be unrestricted, **the community will not allow them to be re-integrated.** The **invisibility** of the badged tournament restriction is not a positive part of the punishment, as that very invisibility prevents the community from seeing and understanding the magnitude of the punishment.   

It is understandable that this is a difficult situation, but that is not an excuse to reduce the consequences of serious rule-breaking, while less dire cases are being denied on the basis that they have not came completely clean. Even if these cases are denied for perfectly valid reasons, the leniency of certain other cases will garner distrust in the support and hostility towards the unrestricted player.

_Clarification_: I have been informed that my suggestion with the restriction length is misinterpretable. To put it simply, the first cheated score would start the timer. In case a rule-breaker would stay under the radar and avoid restriction, this timer would pressure him to come forward. Staying under the radar would increase the punishment, which is appropriate, considering that this player is deceiving the entire community with cheated scores.

**Comment by sr229** [comment]
In my opinion in a community context, marking players visibly is a type of shaming, which is ethically questionable motive. Just because they cheated, we deserved to force it in front of people - they're still human after all, and we need to give them a fair amount of respect as a human, if not, as a community member. I would also like to add *no game* in my knowledge has ever shamed a cheater for ineligibility for competitive play.

For tourneys, I believe a record of the player would be requested by the tourney team to the osu! staff as a screening tool. 

And to complement with @Ephemeralis's sentiment on the restriction time, I also believe increasing the time has no productive gain from it, as this would either end up in the following consequences, as stated by Ephemeral himself:

- Restriction Evasion via Alting.
- Using of penetration tools to attempt to hack the system, usually in a DoS fashion.

>Please note that the incrementally increasing punishment would create an incentive for people to come forward as soon as possible, which could actually remove some burden from the support infrastructure. Some of these players may never come forward without this incentive. It creates a sense of urgency, which the current system does not have.

While I agree with this, I do not really think the "sense of urgency" plays here, rather but the concept of conscience, as it plays on "what should I do to prevent this again?", instead of "I need to stop soon".


> The hostility stem from the extremely lenient sentencing of high profile, serious rule-breakers who managed to follow the appeal protocol (which is not very difficult if every single one of their offenses are publicly scrutinised). This had led to players reacting to any high profile restrictions with "see you in 4 months", which clearly shows that even if these players will be unrestricted, the community will not allow them to be re-integrated. The invisibility of the badged tournament restriction is not a positive part of the punishment, as that very invisibility prevents the community from seeing and understanding the magnitude of the punishment.

Keep in mind a violation of any of the rules of the game should have made them ineligible for *any* Tourneys because of their record. I have zero knowledge on how Tourneys work, but I'm pretty sure they had a screening for all participants, hence the people should automatically know that if you have a major violation, you can no longer partake on competitive play.

>At that point, tying restriction length to the first cheated score does not seem to put any strain onto the system, as the support most likely possess the information.

It's not the system but how much the Support team can also manage tickets like this. Sure, the system can handle it, but at the same time, when there's a lot of tickets for appeals and reports, there will always come a time the teams themselves finds their own hands full.


I do not really understand the point of this, as this is more of a botched version of inciting FUD - which 95% doesn't work.

**Comment by sr229** [comment]
I also want to point out that to improve how strict we can go, any major violations should be treated as a "no-appeal" when done for a second time. The user should be barred from appeals. More grave violations should be a "no-appeal" in immediate form.

**Comment by Kverto** [comment]
> they're still human after all

They seriously broke to rules and they need to suffer the equivalent consequences. You are making it sound like a visible sign would be a cruel and unusual punishment. However, if someone cheated and they are granted another chance, there is nothing unusual about marking them until they regain their tournament privileges. It would lift a considerable burden from tournament organisers if they knew who they cannot allow in the tournament without handing in the registered player list after the registration period. Besides, most people already know that a player had been unrestricted recently due to their rank graph. **Giving them a small, unassuming visible mark would avoid any confusion, serve as a tool for tournament organisers and remind everyone that their actions have consequences.**

> I would also like to add no game in my knowledge has ever shamed a cheater for ineligibility for competitive play.

[Just an entire platform](https://puu.sh/CCB0V/0ee76b02a7.jpg)

> I also believe increasing the time has no productive gain from it

You could argue that we should not restrict anyone because they may avoid restriction or retaliate. It is an argument you can use endlessly until there are absolutely no consequences for any rule-breaking. 

The most important thing in restriction is to be as consistently fair as possible, which cannot be achieved under the current system. My proposal would clearly spell out what you need to do to get minimal punishment and if you do not comply, the punishment will increase accordingly. There would be no issue with minor cases receiving the same amount of restriction as calculated rule brokers who deceived the community for months. This would satiate the community by giving out proportionate punishments, make the rule-breakers content, since their punishment would be loud and clear and it would alleviate some of the burden from the support team by making a considerable part of rule-breakers come forward by themselves. 

> but the concept of conscience

There are many different aspects in play why rule breakers come forward. It is too much of a generalisation to say that conscience is the leading cause, especially when it is incredibly easy to fake being sorry for a more lenient punishment. It is even required when they need to hand in the appeals. On top of that, many players would not come forward exactly because of their conscience - they are too ashamed to, and without any incentive, they may just put it off indefinitely. Whatever the cause is, giving rule-breakers an incentive to come forward as soon as possible directly addresses the issue, regardless of the motivation.

In short, the rework would prevent a lot of first time offenders from rule-breaking, since the consequences would be clear and visible. Restriction would be taken much more seriously, which would help unrestricted players to re-integrate into the community - the community would not feel much animosity towards him/her due to receiving a lenient punishment. Most rule-breakers would receive the same amount of punishment as they do now and only who actively avoid punishment would get more. The mark would help tournament organisers filter out ineligible players themselves without the help of the support team.

**Comment by cl8n** [comment]
the "mark" sounds like an okay idea to me since it doesn't imply that the user is *still* untrustworthy, it's just that they were in the past

I think an appropriate display for this would be a permanent entry in the "Account History" section of profiles

**Comment by WalterToro** [comment]
> the "mark" sounds like an okay idea to me since it doesn't imply that the user is _still_ untrustworthy, it's just that they were in the past
> 
> I think an appropriate display for this would be a permanent entry in the "Account History" section of profiles

@cl8n Something like this?

![](https://user-images.githubusercontent.com/3632750/36294591-2cea4d10-1325-11e8-891d-116833b228e0.png)

[Taken from Issue #2384 on osu-web](https://github.com/ppy/osu-web/issues/2384#issuecomment-366143469)

I agree with doing something like this, at least to help with tournament screenings.

**Comment by sr229** [comment]
@WalterToro I think this is considerably more appropriate compared to what @Kverto is proposing, seems its not really, in UX terms, appropriate to block the avatar. We are just indicating the player is ineligible, not to the point we need to obstruct and shove it directly to the face of someone.

**Comment by sr229** [comment]
>The most important thing in restriction is to be as consistently fair as possible, which cannot be achieved under the current system. My proposal would clearly spell out what you need to do to get minimal punishment and if you do not comply, the punishment will increase accordingly. There would be no issue with minor cases receiving the same amount of restriction as calculated rule brokers who deceived the community for months. This would satiate the community by giving out proportionate punishments, make the rule-breakers content, since their punishment would be loud and clear and it would alleviate some of the burden from the support team by making a considerable part of rule-breakers come forward by themselves.

I don't think making lengths farther than it needs to be is considered fair? We can do more alternatives than increasing timeouts for the fact there is *really* no gain on increased timeout lengths other than just giving time for people to plot a more obtrusive way to get access, in most ways, illegally. Fair for you, unfair for the system.

**Comment by Kverto** [comment]
> seems its not really, in UX terms, appropriate to block the avatar. 

Please note that I've considerably softened down my proposal compared to what I presented in the video. I even pointed it out in the last paragraph before the tl;dr.

**Comment by cl8n** [comment]
updated in #2310 , if you have any problems with the current article open a new issue
